store/v2/root/config.go

```diff
@@ -0,0 +1,14 @@
+package root
+
+func DefaultConfig() *Config {
+	return &Config{
+		AppDBBackend: "goleveldb",
+		Options:      DefaultStoreOptions(),
+	}
+}
+
+type Config struct {
+	Home         string
```

Comment:
Let's add the correct mapstructure and toml annotation to prevent this to be written to the config file. And maybe adding a comment on why.

Reply:
Do you know how to do this? The docs seem to indicate that untagged fields will be silently ignored?
https://pkg.go.dev/github.com/mitchellh/mapstructure#hdr-Remainder_Values

Reply:
I guess we only care about the toml annotation then, I suppose it is like json with a `-`